package.json

```diff
@@ -1,27 +1,27 @@
 {
-	"name": "metus-ingest",
-	"version": "1.2.2",
-	"api_version": "1.0.0",
-	"keywords": [
-		"Protocol",
-		"Software"
-	],
-	"manufacturer": "Metus Technology",
-	"product": "Ingest",
-	"shortname": "Metus Ingest",
-	"description": "Ingest plugin for Companion",
-	"main": "index.js",
-	"scripts": {
-		"test": "echo \"Error: no test specified\" && exit 1"
-	},
-	"author": "Jeffrey Davidsz <jeffrey.davidsz@vicreo.eu>",
-	"license": "MIT",
-	"homepage": "https://github.com/bitfocus/companion-module-metus-ingest#readme",
-	"bugs": {
-		"url": "https://github.com/bitfocus/companion-module-metus-ingest/issues"
-	},
-	"repository": {
-		"type": "git",
-		"url": "git+https://github.com/bitfocus/companion-module-metus-ingest.git"
-	}
+  "name": "ingest-controller",
```

Comment:
I thought, by convention, this matched part of the repo name, but I've no idea if that matters or is just usual?

Reply:
This name doesn't really matter anymore, its just convention at this point. Pre 3.0 it mattered